core/rfb.js

```diff
@@ -2115,6 +2128,9 @@ export default class RFB extends EventTargetMixin {
         encs.push(encodings.encodingCopyRect);
         // Only supported with full depth support
         if (this._fbDepth == 24) {
+            if (H264Decoder.check()) {
+                encs.push(encodings.encodingH264);
+            }
```

Comment:
We also need to check that the H.264 encoding is supported by the browser. That is far from universal.